src/main/java/org/alliancegenome/curation_api/jobs/executors/ConstructExecutor.java

```diff
@@ -24,42 +24,45 @@
 @ApplicationScoped
 public class ConstructExecutor extends LoadFileExecutor {
 
-	@Inject
-	ConstructDAO constructDAO;
+	@Inject ConstructDAO constructDAO;
 
-	@Inject
-	ConstructService constructService;
-	
-	@Inject
-	NcbiTaxonTermService ncbiTaxonTermService;
+	@Inject ConstructService constructService;
+
+	@Inject NcbiTaxonTermService ncbiTaxonTermService;
 
 	public void execLoad(BulkLoadFile bulkLoadFile, Boolean cleanUp) {
 
 		BulkManualLoad manual = (BulkManualLoad) bulkLoadFile.getBulkLoad();
 		Log.info("Running with: " + manual.getDataProvider().name());
 
 		IngestDTO ingestDto = readIngestFile(bulkLoadFile, ConstructDTO.class);
-		if (ingestDto == null) return;
-		
+		if (ingestDto == null) {
+			return;
+		}
+
 		List<ConstructDTO> constructs = ingestDto.getConstructIngestSet();
-		if (constructs == null) constructs = new ArrayList<>();
-		
+		if (constructs == null) {
+			constructs = new ArrayList<>();
+		}
+
 		BackendBulkDataProvider dataProvider = manual.getDataProvider();
-		
+
 		List<Long> constructIdsLoaded = new ArrayList<>();
 		List<Long> constructIdsBefore = new ArrayList<>();
 		if (cleanUp) {
 			constructIdsBefore.addAll(constructService.getConstructIdsByDataProvider(dataProvider));
 			Log.debug("runLoad: Before: total " + constructIdsBefore.size());
 		}
-		
+
 		bulkLoadFile.setRecordCount(constructs.size() + bulkLoadFile.getRecordCount());
 		bulkLoadFileDAO.merge(bulkLoadFile);
-		
+
 		BulkLoadFileHistory history = new BulkLoadFileHistory(constructs.size());
 		createHistory(history, bulkLoadFile);
 		boolean success = runLoad(constructService, history, dataProvider, constructs, constructIdsLoaded);
-		if(success && cleanUp) runCleanup(constructService, history, dataProvider.name(), constructIdsBefore, constructIdsLoaded, bulkLoadFile.getMd5Sum());
+		if(success && cleanUp) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.